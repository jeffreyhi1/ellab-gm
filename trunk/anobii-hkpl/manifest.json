{
  "name": "Bookworm",
  "version": "6",
  "manifest_version": 2,
  "description": "Integrate aNobii, Hong Kong Public Library and books.com.tw. Features like searching Hong Kong Public Library in aNobii pages.",
  "icons": {
    "16": "icon-16.png",
    "48": "icon-48.png",
    "128": "icon-128.png"
  },
  "content_scripts": [
    {
      "matches": ["http://www.anobii.com/books/*", "http://www.anobii.com/wishlist*",
                  "http://www.anobii.com/*/books*", "http://www.anobii.com/*/wishlist*",
                  "http://www.anobii.com/search*", "http://www.anobii.com/contributors/*",
                  "http://www.anobii.com/tags/*", "http://www.anobii.com/news_neighbor*",
                  "http://webcat.hkpl.gov.hk/*",
                  "https://webcat.hkpl.gov.hk/*",
                  "https://www.hkpl.gov.hk/tc_chi/collections/collections_bs/collections_bs.html*",
                  "http://www.books.com.tw/exep/prod/booksfile.php?item=*"],
      "css": ["anobiihkpl.css"],
      "js": ["ellab-utils.js", "anobiihkpl.user.js"]
    }
  ],
  "minimum_chrome_version": "20",
  "permissions": [
    "http://webcat.hkpl.gov.hk/", "http://iapp2.anobii.com/", "http://www.books.com.tw/, http://api.douban.com/"
  ],
  "web_accessible_resources": ["loading.gif", "shadowAlpha.png"]
}