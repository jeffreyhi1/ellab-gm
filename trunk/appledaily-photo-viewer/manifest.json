{
  "name": "Appledaily Large Photo Viewer",
  "version": "3",
  "description": "蘋果日報及壹傳媒改版後用 Flash 來顯示放大圖片，不利大家儲存和轉載。這個 extension 會在版面增加一個圖示指向真正的大圖片。",
  "content_scripts": [
    {
      "matches": ["http://*.nextmedia.com/template/*", "http://*.nextmedia.com/realtime/*"],
      "js": ["appledailylargephotoviewer.user.js"]
    }
  ],
  "icons": {
    "16": "icon-16.png",
    "48": "icon-48.png",
    "128": "icon-128.png"
  }
}