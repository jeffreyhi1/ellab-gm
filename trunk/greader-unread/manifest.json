{
  "name": "Google Unread Count",
  "version": "7",
  "description": "Display actual unread count instead of '1000+' in Google Reader.",
  "content_scripts": [
    {
      "matches": ["http://www.google.com/reader/*", "https://www.google.com/reader/*"],
      "js": ["googlereaderunreadcount.user.js"]
    }
  ]
}