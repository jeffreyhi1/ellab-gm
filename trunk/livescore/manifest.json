{
  "name": "Better Livescore",
  "version": "2",
  "manifest_version": 2,
  "description": "Make livescore.com better. Show the match details in the same page instead of pop-up and show the match time in your local time",
  "icons": {
    "16": "icon-16.png",
    "48": "icon-48.png",
    "128": "icon-128.png"
  },
  "content_scripts": [
    {
      "matches": ["http://www.livescore.com/", "http://www.livescores.com/*", "http://www.livescore.co.uk/*",
                  "http://www.livescore.com/soccer/*"],
      "run_at": "document_end",
      "js": ["jquery-1.8.2.min.js", "jquery.color-2.1.0.min.js", "livescore.user.js"]
    }
  ],
  "web_accessible_resources": ["loading.gif"]
}
