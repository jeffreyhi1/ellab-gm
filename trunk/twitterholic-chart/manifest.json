{
  "name": "Twitterholic Chart",
  "version": "1",
  "description": "Show a more detail chart in twitterholic.com user stat page",
  "content_scripts": [
    {
      "matches": ["http://twitterholic.com/*"],
      "js": ["twitterholicchart.user.js"]
    }
  ],
  "icons": {
    "48": "icon-48.png"
  },
  "update_url": "http://ellab-gm.googlecode.com/svn/updates/twitterholic-chart-crx.xml"
}