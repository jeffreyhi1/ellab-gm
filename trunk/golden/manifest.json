{
  "name": "Golden",
  "version": "1",
  "manifest_version": 2,
  "description": "改善香港高登 hkgolden.com 的瀏覽體驗，包括精簡版面和其他貼心功能。",
  "icons": {
    "16": "images/icon-16.png",
    "48": "images/icon-48.png",
    "128": "images/icon-128.png"
  },
  "background": { "scripts": ["background.js"] },
  "content_scripts": [
    {
      "matches": ["http://*.hkgolden.com/*"],
      "all_frames": true,
      "run_at": "document_end",
      "js": ["js/sizzle-1.8.2.js", "js/moment-1.7.2.min.js", "js/moment-lang-zh-tw.js", "ellab-utils-6.js", "golden.user.js"],
      "css": ["golden.css"]
    }
  ],
  "options_page": "options.html",
  "content_security_policy": "script-src 'self' https://use.edgefonts.net/; object-src 'self'",
  "permissions": [ "storage", "tabs", "http://use.edgefonts.net/league-gothic.js" ],
  "web_accessible_resources": ["images/clock.png",
                               "images/golden-favicon.png", "images/golden-favicon-blank.png",
                               "images/new-message.png", "images/new-message-blank.png"]
}
